server/docs/orders/specificOrder.js

```diff
@@ -0,0 +1,125 @@
+module.exports = {

```

Comment:
I like that you do docs.
What is your experience with making APIs?
Also public/authed APIs for clients?

Reply:
Thanks. I have built a number.

Reply:
Any longer comments or thoughts on it :D Or showcasing a particular build?

Reply:
Some of the APIs i have built include:

- An SMS management API that modeled contacts ( name of person, phone number of person) and SMS (person sending sms, person receiving sms, message of sms, sms status).

- A population management API that modeled the following abstractions: User (email, password), Location (name, number of male residents, number of female residents and the total number of residents) 

- Wger, a workout manager that manages exercises and personal workouts.